ui/raidboss/emulator/overrides/RaidEmulatorTimeline.js

```diff
@@ -21,24 +16,21 @@ export default class RaidEmulatorTimeline extends Timeline {
     });
   }
 
-  emulatedSync(timestampOffset) {
-    if (!this.emulatedFightSyncLastOffset)
+  emulatedSync(currentTimestamp) {
+    if (!currentTimestamp)
       return;
 
-    this.SyncTo(this.emulatedFightSync +
-      ((timestampOffset - this.emulatedFightSyncLastOffset) / 1000), timestampOffset);
-    this._OnUpdateTimer(timestampOffset);
-  }
+    // This is a bit complicated due to jumps in timelines. If we've already got a timebase,
+    // fightNow needs to be calculated based off of that instead of engageAt
+    // timebase = 0 when not set
+    const baseTimestamp = this.timebase || this.emulator.currentEncounter.encounter.engageAt;
+    const fightNow = (currentTimestamp - baseTimestamp) / 1000;
```

Comment:
This does make me wonder if `this.timebase` should be `null` instead of `0` when stopped.  Not a thing to fix in this patch, but this took a little bit of staring at to remember that `this.timebase` is a "real" time (and not an offset) that is set to zero when unset.

Reply:
Yeah, I scratched my head a bit at that. I can look into a PR for it after this one is merged?

Reply:
Sounds good!